### PR TITLE
New Function

### PR DESCRIPTION
**Pull request by MonteiroCM**
Created the possibility of making a total per page.

### CODE REVIEW

File: src/app/Classes/Attributes/Meta.php [hunk 1]
```diff
@@ -7,6 +7,6 @@ class Meta
     const List = [
         'searchable', 'sortable', 'sort:ASC', 'sort:DESC', 'translatable',
         'boolean', 'slot', 'rogue', 'total', 'date', 'icon', 'clickable',
-        'customTotal', 'notExportable', 'nullLast',
+        'customTotal', 'notExportable', 'nullLast', 'totalPage',
```
[the remark this answers is not in the document]
Comment (MonteiroCM) [reply src/app/Classes/Attributes/Meta.php]: ok

File: src/resources/js/components/enso/vuedatatable/VueTable.vue [hunk 2]
```diff
@@ -59,9 +60,7 @@
                         v-if="visibleColumns[i].meta.customTotal">
                         <slot :name="`${visibleColumns[i].name}_custom_total`"
                             :total="body ? body.total : []"
-                            :column="visibleColumns[i]">
-                            {{ `${visibleColumns[i].name}_custom_total` }}
-                        </slot>
+                            :column="visibleColumns[i]">{{ `${visibleColumns[i].name}_custom_total` }}</slot>
```
[the remark this answers is not in the document]
Comment (MonteiroCM) [reply src/resources/js/components/enso/vuedatatable/VueTable.vue]: Ignore that

**Comment by aocneanu** [comment]
@MonteiroCM ?

**Comment by MonteiroCM** [comment]
@aocneanu I had problems at work and I had to take a break, but I'll take this week and do it.

**Comment by aocneanu** [comment]
No problem, thanks again.

**Comment by aocneanu** [comment]
Closing due to lack of activity

**Pull request closed without merging**